src/urllib3/connectionpool.py

```diff
@@ -391,6 +390,19 @@ def _make_request(
             :class:`urllib3.util.Timeout`, which gives you more fine-grained
             control over your timeouts.
         """
+
+        # This value should already be assigned to the `url` parameter
+        if "request_url" in httplib_request_kw:
```

Comment:
We shouldn't need to modify `httplib_request_kw` at this point, can we make all parameters explicit and avoid a `**` at all in our method signature?

Reply:
done